simapp/helpers/test_helpers.go

```diff
@@ -18,8 +18,8 @@ const (
 	SimAppChainID   = "simulation-app"
 )
 
-// GenTx generates a signed mock transaction.
-func GenTx(gen client.TxConfig, msgs []sdk.Msg, feeAmt sdk.Coins, gas uint64, chainID string, accNums, accSeqs []uint64, priv ...cryptotypes.PrivKey) (sdk.Tx, error) {
+// GenerateTx generates a signed mock transaction.
+func GenerateTx(gen client.TxConfig, msgs []sdk.Msg, feeAmt sdk.Coins, gas uint64, chainID string, accNums, accSeqs []uint64, priv ...cryptotypes.PrivKey) (sdk.Tx, error) {
```

Comment:
why we are changing this? GenTx is widely used in the SDK.

Reply:
I think `GenTx` is okay here.